Running flake8 in production

Running flake8 in production `build.nix` seems not a good idea since new warnings will stop the compilation.

```
/nix/store/9hcqavcpv22ddbcqd6cn8crj8pn6b4mr-src/arguments.py:29:9: F841 local variable 'e' is assigned to but never used
```

--- a/fetch-git/build.nix
+++ b/fetch-git/build.nix
@@ -15,5 +15,5 @@ runCommand "fetch-git" {
 
     patchShebangs $out/lib/fetch-git.py
     wrapProgram $out/bin/fetch-git \
-      --set PATH "${git}/bin:${gnutar}/bin:${bash}/bin"
+      --set PATH "${python3}/bin:${git}/bin:${gnutar}/bin:${bash}/bin"
 ''

--- a/fetch-git/build.nix
+++ b/fetch-git/build.nix
@@ -3,11 +3,9 @@
 runCommand "fetch-git" {
   src = ./src;
   inherit git gnutar python3;
-  nativeBuildInputs = [ makeWrapper python3Packages.flake8 ];
+  nativeBuildInputs = [ makeWrapper ];
 }
 ''
-    flake8 $src/*.py
-
     mkdir -p $out/bin $out/lib
 
     cp $src/*.py $out/lib

